content/docs/avalanche-l1s/deploy-a-avalanche-l1/avalanche-mainnet.mdx

```diff
@@ -130,341 +118,133 @@ After that, CLI shows the `Mainnet` Ledger address used to fund the deployment:
++--------------------+----------------------------------------------------+
+| Chain Name         | testblockchain                                     |
++--------------------+----------------------------------------------------+
+| Subnet ID          | 2cNuyBhvAd4jH5bFSGndezhB66Z4UHYAsLCMGoCpvhXVhrZfgd |
++--------------------+----------------------------------------------------+
+| VM ID              | qcvkEX1zWSz7PtGd7CKvPRBqLVTzA7qyMPvkh5NMDWkuhrcCu  |
++--------------------+----------------------------------------------------+
+| Blockchain ID      | 2U7vNdB78xTiN6QtZ9aetfKoGtQhfeEPQG6QZC8bpq8QMf4cDx |
++--------------------+                                                    +
+| P-Chain TXID       |                                                    |
++--------------------+----------------------------------------------------+
 ```
 
-#### Restart the Node[​](#restart-the-node "Direct link to heading")
-
-It's **required to restart the node**.
-
-### Setup Node Manually[​](#setup-node-manually "Direct link to heading")
-
-By choosing "Manual" instead, the tool prints _instructions_. The user is going to have to follow these instructions and apply them to the node. Note that the IDs for the VM and Avalanche L1 are different for each Avalanche L1 deployment and you shouldn't copy them from this tutorial.
-
-```bash
-✔ Manual
-
-To setup your node, you must do two things:
-
-1. Add your VM binary to your node's plugin directory
-2. Update your node config to start validating the Avalanche L1
-
-To add the VM to your plugin directory, copy or scp from /tmp/tGBrMADESojmu5Et9CpbGCrmVf9fiAJtZM5ZJ3YVDj5JTu2qw
-
-If you installed avalanchego manually, your plugin directory is likely
-avalanchego/build/plugins.
-
-If you start your node from the command line WITHOUT a config file (e.g. via command
-line or systemd script), add the following flag to your node's startup command:
-
---track-subnets=2b175hLJhGdj3CzgXENso9CmwMgejaCQXhMFzBsm8hXbH2MF7H
-(if the node already has a track-subnets config, append the new value by
-comma-separating it).
-
-For example:
-./build/avalanchego --network-id=Mainnet --track-subnets=2b175hLJhGdj3CzgXENso9CmwMgejaCQXhMFzBsm8hXbH2MF7H
-
-If you start the node via a JSON config file, add this to your config file:
-track-subnets: 2b175hLJhGdj3CzgXENso9CmwMgejaCQXhMFzBsm8hXbH2MF7H
-
-TIP: Try this command with the --avalanchego-config flag pointing to your config file,
-this tool is going to try to update the file automatically (make sure it can write to it).
-
-After you update your config, you are going to need to restart your node for the changes to
-take effect.
-```
+To get your new Avalanche L1 information, visit the [Avalanche L1 Explorer](https://subnets-test.avax.network/). The search best works by blockchain ID, so in this example, enter `2U7vNdB78xTiN6QtZ9aetfKoGtQhfeEPQG6QZC8bpq8QMf4cDx` into the search box and you should see your shiny new blockchain information.
```

Comment:
We should also include `avalanche blockchain describe` as an option for printing blockchain information